streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.java

```diff
@@ -249,7 +251,7 @@ public int maxFileOpeningThreads() {
 
     @Override
     public Options setMaxTotalWalSize(final long maxTotalWalSize) {
-        dbOptions.setMaxTotalWalSize(maxTotalWalSize);
+        LOGGER.warn("WAL is explicitly disabled by Streams in RocksDB. Setting option 'maxTotalWalSize' will be ignored");
```

Comment:
Could you please add tests that verify the log messages? You can find an example how to verify log messages in `KTableSourceTest#kTableShouldLogAndMeterOnSkippedRecords()`.

Reply:
@cadonna 
Sorry for the late reply, was on vacation.
I added the rest of the wal options to the test.